docs/developer/app-store/apps/cms.mdx

```diff
@@ -0,0 +1,183 @@
+
+Once the connection is configured, the app will allow you to use this connection and scan the entire Saleor database for products and send them to the provider.
+
+:::caution Limitation
+At the moment, the App is performing bulk sync on the frontend. That means when this is happening, the App must stay open. Closing the window will stop the operation.
+:::
+
+## Configuration
+
+To configure the app, follow the instructions built into the app.
+
+Usually, you will need to perform the following steps:
+
+1. Create a provider configuration for the chosen provider.
+2. Create a connection of this provider to the selected channel.
+3. Perform an initial sync.
+4. Edit or create some product variant to see if it is synchronized with the provider.
+
+### CMS fields setup
+
+The App allows you to map Saleor product fields to the CMS fields.
+Some providers allow the App to autocomplete available fields. If autocomplete is not available, you must ensure fields are entered without typos.
+
+## Providers
+
+### Contentful
+
+- Fields are autocompleted; you must provide [space ID](https://www.contentful.com/help/find-space-id/) and a [token](https://www.contentful.com/help/personal-access-tokens/).
+- Entries will be created with an ID that matches Saleor product variant ID.
+- Bulk sync operations are limited to 2 items per second due to Contentful API limitations.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+### DatoCMS
+
+- Fields are autocompleted. You must provide a full-access API token (Content Management API).
+- Entries will be created with IDs generated by DatoCMS. You should set the variant ID field to be unique.
+- Bulk sync operations will be rate-limited by DatoCMS client itself.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+### Strapi
+
+- Fields are not autocompleted. You must provide an auth token with full access to the product variant entity and API base URL (without `/api` suffix).
+- Item type must be plural, eg, `saleor-products` instead `saleor-product`.
+- You must ensure field mapping is correct because there is no autocomplete.
+- All fields must be the type of `string/text`, except `channels`, which should be type `json`.
+
+## General limitations and assumptions
+
+- App is unidirectional. Changes in CMS will not be reflected in Saleor.
+- Bulk / initial sync runs only on the frontend. This is usually a one-time operation, but the browser must be opened. In case of a connection problem, the operation can fail.
+- If **UNIQUE** attribute is not set on the variant ID field in the CMS, there will be no guarantee that duplicates will not be created.
+- It's normal to see console network errors during bulk sync. The app is usually trying to create an item in the CMS. Once it faces conflict or rate limit error, it will retry.
+
+## Development
```

Comment:
**suggestion (blocking)**:

"Development" should go before "Limitations", just like in "Emails and Messages" docs.

Reply:
I see. In that case, I would put "Limitations" just below the "Assumptions". This way, we give an overview of what the app is and is not right at the beginning.